MANIFEST.in

```diff
@@ -1,6 +1,6 @@
 include mache/cime_machine_config/*.xml
 include mache/machines/*.cfg
-include mache/spack/*.yaml
-include mache/spack/*.template
-include mache/spack/*.sh
-include mache/spack/*.csh
+include mache/spack/templates/*.yaml
+include mache/spack/templates/*.template
+include mache/spack/templates/*.sh
+include mache/spack/templates/*.csh
```

Comment:
Thanks for catching that one!  It wouldn't otherwise show up until we do a release and build a conda package, since the files are just there in the branch when you do local testing.